src/ext.js

```diff
@@ -241,7 +255,64 @@ export default function ext({ translator, shouldHide, senseNavigation }) {
                 automationTriggeredText: {
                   label: `Object.ActionButton.Automation.RunModeTriggeredHelp`,
                   component: 'text',
-                  show: (data) => checkShowAction(data, 'automation') && multiUserAutomation && data.automationShowTriggered,
+                  show: (data) => checkShowAction(data, 'automation') && multiUserAutomation && data.automationShowTriggered && data.automationTriggered,
+                },
+                showNotification: {
+                  ref: 'showNotification',
+                  type: 'boolean',
+                  translation: 'Object.ActionButton.Automation.ShowNotification',
+                  component: 'switch',
+                  options: [
+                    {
+                      value: true,
+                      translation: 'properties.on',
+                    },
+                    {
+                      value: false,
+                      translation: 'properties.off',
+                    },
+                  ],
+                  defaultValue: false,
+                  show: (data) => checkShowAction(data, 'automation') && multiUserAutomation
+                },
+                notificationDuration: {
+                  type: 'number',
+                  ref: 'notificationDuration',
+                  translation: 'Object.ActionButton.Automation.NotificationDuration',
+                  defaultValue: 4,
+                  expression: 'optional',
+                  show: (data) => checkShowAction(data, 'automation') && data.showNotification && multiUserAutomation
+                },
```

Comment:
It is needed to be variable in this case as depending on which automation the end user is triggering we could require a short duration (user just triggers some process and needs to be alerted it ran ok) or it could require that the user interact with the popup (automation generates a report and sends them a link through the popup which the user needs to click on...in this case it would make more sense to have a longer timeout)